conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/ethereum/EthereumDataEndpointsCreator.scala

```diff
@@ -0,0 +1,63 @@
+package tech.cryptonomic.conseil.api.routes.platform.data.ethereum
+
+import tech.cryptonomic.conseil.api.routes.platform.data.{ApiDataEndpoints, ApiDataJsonSchemas}
+import tech.cryptonomic.conseil.common.generic.chain.DataTypes.QueryResponse
+
+/** Trait containing endpoints definitions for Ethereum-related blockchains */
+trait EthereumDataEndpointsCreator extends ApiDataEndpoints with ApiDataJsonSchemas with EthereumFilterFromQueryString {
+
+  /** V2 Blocks endpoint definition */
+  private[ethereum] def blocksEndpoint(root: Path[String]) : Endpoint[((String, EthereumFilter), Option[String]), Option[
+    List[QueryResponse]
+  ]] =
+    endpoint(
+      request = get(url = root / "blocks" /? ethereumQsFilter, headers = optHeader("apiKey")),
+      response = compatibilityQuery[List[QueryResponse]]("blocks"),
+      tags = List("Blocks")
```

Comment:
as we add new chains I think we should include the name of the chain in the tags

Reply:
Would you like me to add that to all of the endpoints?

Reply:
Well, did that for all of them :)